src/hooks/constants/lib/fetchConstants.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiPromise } from '@polkadot/api';
+import { Constants, Fee } from '../../../generated/graphql';
+
+export const repayFeeDataType = '(u32, u32)';
+
+export const unwrapRepayFeeToFee = (apiInstance: ApiPromise): Fee => {
+  const apiRepayFee = apiInstance
+    .createType(repayFeeDataType, apiInstance.consts.lbp.getRepayFee)
+    .toHuman() as Array<string>;
+
+  return {
+    numerator: apiRepayFee[0],
+    denominator: apiRepayFee[1],
+  };
+};
+export const fetchConstants = (apiInstance: ApiPromise): Constants => ({
+  lbp: {
+    repayFee: unwrapRepayFeeToFee(apiInstance),
```

Comment:
lbp constants should be a sub resolver for the abstract `constants` entity, that does not have a resolver of its own. This will allow for constants composability.

Reply:
analysis+solution:  https://github.com/galacticcouncil/Basilisk-ui/pull/109/files#r810244232